fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -478,7 +478,8 @@ private FHIRRestOperationResponse doPatchOrUpdate(String type, String id, FHIRPa
             if (ior.getPrevResource() != null) {
                 performVersionAwareUpdateCheck(ior.getPrevResource(), ifMatchValue);
 
-                if (skippableUpdate && !isDeleted) {
+                // In the case of a patch, we should not be updating meaninglessly.
+                if ((skippableUpdate || patch != null) && !isDeleted) {
```

Comment:
I wonder if we should eventually make skippableUpdate the default (instead of forcing clients to opt-in)

Reply:
It's probably the preferred way to do it - definitely reduces burden.